src/client/conn.rs

```diff
@@ -87,7 +87,8 @@ where
     H2(#[pin] proto::h2::ClientTask<B>, PhantomData<fn(T)>),
 }
 
-/// Returns a handshake future over some IO.
+/// Returns a handshake future over some IO. The io parameter will usually be a
+/// [tokio::net::TcpStream](https://docs.rs/tokio/1.6.1/tokio/net/struct.TcpStream.html).
```

Comment:
I wonder, so as to not encode the version directly into the source, can we just point at `(tokio::net::TcpStream)`? Also, I'd put the second sentence in a new line / new paragraph.